multi-tenancy/schema-isolation/standalone.xml

```diff
@@ -132,10 +131,10 @@
                 </handlers>
             </root-logger>
             <formatter name="PATTERN">
-                <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %X %s%e%n"/>
+                <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
             </formatter>
             <formatter name="COLOR-PATTERN">
-                <pattern-formatter pattern="%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %X %s%e%n"/>
+                <pattern-formatter pattern="%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
```

Comment:
Those changes are not WildFly update related, I guess?

Reply:
If you agree, we can also revert that in the other `standalone.xml`

Reply:
Well, they are Wildfly 19 related. All changes I included are part of the standalone.xml we ship with 7.15 and wildfly 22. I think it makes sense to stick as close as possible to that config, as it is meant to be a drop in replacement.